client/src/js/samples/components/__tests__/General.test.js

```diff
@@ -14,16 +14,10 @@ describe("<SampleDetailGeneral />", () => {
             locale: "Bar",
             paired: false,
             subtractionId: "Arabidopsis thaliana",
-            srna: false
+            libraryType: "sRNA"
```

Comment:
Allowed values of `libraryType` are `normal`, `srna`, and `amplicon`. Remember you need to convert these lowercase strings to human-friendly words.

You should do this using an object defined at the top of the module:
```js
const libraryTypes = {
  normal: "Normal",
  srna: "sRNA",
  amplicon: "Amplicon"
};
```
Then in the component, convert the lowercase value to human-friendly:
```js
libraryType: libraryTypes[library_type]
```
You need test cases for each allowed value to make sure the word is formatted correctly.